xml/ha_config_cli.xml

```diff
@@ -1775,13 +1774,13 @@ hostname (string): Hostname
       </para>
 <screen>&prompt.crm.conf;<command>primitive</command> my-rsc ocf:myCorp:myAppl \
     op monitor interval=60 \
-    op monitor interval=61 role=Master</screen>
+    op monitor interval=61 role="Promoted</screen>
```

Comment:
We could drop the single quotation mark here :-)

Reply:
Oh dear! Thanks for spotting it! :sweat_smile: